src/main/java/de/numcodex/sq2cql/model/cql/Context.java

```diff
@@ -0,0 +1,30 @@
+package de.numcodex.sq2cql.model.cql;
+
+import static java.util.stream.Collectors.joining;
+
+import de.numcodex.sq2cql.PrintContext;
+import java.util.List;
+
+public record Context(String contextName, List<ExpressionDefinition> expressionDefinitions) {
+
+
+  public Context {
+    expressionDefinitions = List.copyOf(expressionDefinitions);
+  }
+
+  public static Context of() {
+    return new Context("", List.of());
+  }
+
+  public static Context of(String contextName, List<ExpressionDefinition> expressionDefinitions) {
+    return new Context(contextName, expressionDefinitions);
+  }
+
+  public String print(PrintContext printContext) {
```

Comment:
Please remove the print context here and at the library level, because it's always the ZERO print context. We only need the print context at expression level.